tmt/utils.py

```diff
@@ -4832,21 +4837,45 @@ def git_clone(
     """
     depth = ['--depth=1'] if shallow else []
 
+    def get_env(env: str, default_value: Optional[str] = None) -> Optional[int]:
+        """ Get the value of an environment variable and convert it to be integer """
+        out = None
+        value = os.environ.get(env, default_value)
+        if value is not None:
+            try:
+                out = int(value)
+            except ValueError:
+                out = None
+        return out
+
+    retries = retries or get_env('TMT_GIT_CLONE_RETRIES', str(DEFAULT_GIT_CLONE_RETRIES))
+    timeout = timeout or get_env('TMT_GIT_CLONE_TIMEOUT', None)
```

Comment:
Recently a new `tmt.utils.retry()` function has been added, shall we use here as well?

* https://tmt.readthedocs.io/en/latest/autodocs/tmt.html#tmt.utils.retry

If so, let's use the same parameters (e.g. `attempts` and `interval`) to be consistent.

Reply:
Yes, updated to use `retry()`. Please help to take a look again, thanks!

Reply:
Thanks. Why have you introduced `gc_` prefixes for the new parameters?

Reply:
It was because `retry(git_clone, ...)` was called in `git_clone()` itself, the `**kwargs` should not include two identical key-value pairs . Now `gc_` prefixes are removed.